.github/workflows/cd_gestor_entregas.yml

```
@@ -0,0 +1,98 @@
+name: Deploy Gestor Entrega
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'backend/logistica_servicio/gestor_entregas/**'
+      - '.github/workflows/cd_gestor_entregas.yml'
+  workflow_dispatch:
+jobs:
+   deploy_to_gke:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 3

(indentation)